src/message.ts

```diff
@@ -1150,30 +1189,52 @@ export class MediaMessage extends Message {
    *   EMOJI                    = 8,
    * }
    * ```
-   * But, it should be noted that when forwarding ATTACH type message, if the file size is greater than 25Mb, the forwarding will fail.
-   * The reason is that the server limits the forwarding of files above 25Mb. You need to download the file and use `new MediaMessage (file)` to send the file.
+   * It should be noted that when forwarding ATTACH type message, if the file size is greater than 25Mb, the forwarding will fail.
+   * The reason is that the server shields the web wx to download more than 25Mb files with a file size of 0.
+   *
+   * But if the file is uploaded by you using wechaty, you can forward it.
```

Comment:
Note that this is a new feature that supports forwarding your own uploaded 25Mb + files.